app/javascript/custom_scripts/view_more_description.js

```diff
@@ -0,0 +1,27 @@
+function viewMore(){
+  if (document.getElementById('button_for_view_more') !=null){
+    const VIEW_MORE = 'View More'
```

Comment:
nit: Im not sure that constants in js must be in upcase, viewMore
in js - good practice to use camelCase variables

Reply:
By convention, the constant identifiers are in uppercase.ES6